libs/ui-watt/src/lib/components/shell/nav-list/watt-nav-list.component.spec.ts

```diff
@@ -154,11 +156,11 @@ describe(WattNavListModule.name, () => {
       const link = await screen.findByRole('link', {
         name: /energinet/i,
       });
-      expect(link).toHaveAttribute('href', 'http://energinet.dk');
+      expect(link).toHaveAttribute('href', energinetDkUrl);
```

Comment:
issue: This link uses the `http://` protocol so it can't use the same variable as the instances above where the `https://` protocol (notice the `s`) is used.

Reply:
Good catch. Could it be because the changes are only in a spec file, Nx didn't detect the library as effected and didn't run the tests? https://github.com/Energinet-DataHub/greenforce-frontend/runs/5716634156?check_suite_focus=true#step:9:24
Should would've been catched otherwise.